Gemfile.lock

```diff
@@ -113,6 +113,8 @@ GEM
     minitest (5.14.4)
     msgpack (1.4.2)
     nio4r (2.5.7)
+    nokogiri (1.11.5-x86_64-darwin)
```

Comment:
this is the line that gave the error before when merging. anyone know what darwin is?